shared/constants/network.ts

```diff
@@ -1101,6 +1101,11 @@ export const FEATURED_NETWORK_CHAIN_IDS = [
   ...FEATURED_RPCS.map((rpc) => rpc.chainId),
 ];
 
+export const FEATURED_NETWORK_NAMES = [
```

Comment:
appears no longer used in latest commit

Reply:
https://github.com/MetaMask/metamask-extension/pull/29112/commits/a1e51cf820041e446ee6cdf81165698729cfb224